nservicebus/endpoint/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Endpoint
+summary: Describe general concept of Endpoint and Endpoint Instance.
+tags:
+- Endpoint
+- Endpoint Instance
+---
+
```

Comment:
@mikeminutillo  u could make the intro an include and re-use the text from concept-overview.md